### PR TITLE
modules/xbps: fix error message

### PR DESCRIPTION
**Pull request by CameronNemo**
##### SUMMARY

The previous error message was not giving the full or even correct information to the user.

##### ISSUE TYPE

- Bugfix Pull Request

##### COMPONENT NAME

community.general.xbps

### CODE REVIEW

**Comment by the-maldridge** [comment]
LGTM

**Review by felixfontein — COMMENTED** [review]
Thanks for your contribution! Can you please add a [changelog fragment](https://docs.ansible.com/ansible/devel/community/collection_development_process.html#creating-a-changelog-fragment)? Thanks.

**Comment by russoz** [comment]
LGTM, pending changelog fragment

**Comment by CameronNemo** [comment]
alright lmk if that changelog fragment is good

File: changelogs/fragments/4438-fix-error-message.yaml [hunk 1]
```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - xbps - fix error message while installing packages
```
Comment (felixfontein) [inline changelogs/fragments/4438-fix-error-message.yaml:2]:
```suggestion
  - xbps - fix error message that is reported when installing packages fails (https://github.com/ansible-collections/community.general/pull/4438).
```

Comment (CameronNemo) [reply changelogs/fragments/4438-fix-error-message.yaml:2]: alright thanks

**Review by russoz — APPROVED** [review]
LGTM

**Comment by felixfontein** [comment]
@CameronNemo thanks for fixing this!
@russoz thanks for reviewing!

**Pull request merged**